students/ador_yano/Lesson03/slicing_sequence.py

```diff
@@ -0,0 +1,47 @@
+# slicing_sequence.py implements the Lesson 3 - Slicing Lab Exercise assignment from UWPCE Python Programming
+
+intro = '''UWPCE Python Programming: Lesson 3 Assignment -- Slicing Lab Exercise
+Functions that take a sequence as an argument then return a copy of that sequence
+with modifications:
+1. exchange_first_last(seq): the first and last items exchanged
+2. remove_alternate_item(seq): every other item removed
+3. remove_first_last_4_alternate_item(seq): the first 4 and the last 4 items removed, and then every other item in between
+4. reverse_slice(seq): the elements reversed (just with slicing)
+5. mid_last_first(seq): the middle third, then last third, then the first third in the new order
+'''
+print(intro)
+
+def exchange_first_last(seq):
+    return seq[-1:]+seq[1:-1]+seq[:1]
+
+def remove_alternate_item(seq):
+    return seq[0:len(seq)-1:2]
```

Comment:
since you're using default values for start/stop it can be simplified to : `seq[::2]`